### PR TITLE
Fix quotes in annotation string literals

### PR DESCRIPTION
**Pull request by brettkail-wk**
@Workiva/messaging-pp

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Module Findings
No security relevant content detected. Please review for security relevance and request security review as needed.

File: compiler/generator/golang/generator.go [hunk 1]
```diff
@@ -1856,7 +1861,7 @@ func (g *Generator) generateProcessor(service *parser.Service) string {
 		if len(method.Annotations) > 0 {
 			contents += fmt.Sprintf("\tp.AddToAnnotationsMap(\"%s\", map[string]string{\n", methodLower)
 			for _, annotation := range method.Annotations {
-				contents += fmt.Sprintf("\t\t\"%s\": \"%s\",\n", annotation.Name, annotation.Value)
+				contents += fmt.Sprintf("\t\t%s: %s,\n", g.quote(annotation.Name), g.quote(annotation.Value))
```
Comment (brianshannan-wf) [inline compiler/generator/golang/generator.go]: Do we want annotation names to be able to contain quotes? I guess that seems kind of weird to me but I could be convinced

Comment (brettkail-wk) [reply compiler/generator/golang/generator.go]: You're right, annotation keys must be identifiers, so we don't need to quote them.

**Comment by brianshannan-wf** [comment]
+1

**Comment by charliestrawn-wf** [comment]
lgtm

**Pull request merged**